### DISCUSSION TITLE
Default hdr does not work correctly

### DISCUSSION THREAD

**Discussion by DenisAzarkin** [google/model-viewer#3603] [Q&A] [answered]
### Description
Default hdr does not work correctly. Distorts colors and does not rotate, gives hard shadows. The best option for Default hdr is Neutral hdr.
I think hdr Neutral and Default have swapped places. Neutral hdr is installed by default, but it is not. The Default hdr is not set by default, but there is no tag in the code.

Is it possible to make Neutral hdr in place of Default hdr?

![Screenshot_280](https://user-images.githubusercontent.com/23003266/181115391-d8307db5-63a9-4945-a99f-2a84be653d61.png) ![Screenshot_281](https://user-images.githubusercontent.com/23003266/181115403-b70b554f-31c4-4714-b236-06d6ed95bbc0.png)

### Version

- model-viewer: v1.12.0

### Browser Affected

- Chrome
- Firefox

### OS

- Android
- iOS
- Windows

### AR

- WebXR
- SceneViewer
- QuickLook

**Comment by elalish** [comment]
Agreed, it is even in plan: https://github.com/google/model-viewer/discussions/3460

Our editor's defaults are more opinionated, and so tend to lead the actual API's defaults.

**Comment by DenisAzarkin** [comment, marked as answer]
This problem was solved here #3675 Thanks!
